src/PendingObjects/TestCall.php

```diff
@@ -131,10 +131,10 @@ public function group(string ...$groups): TestCall
     /**
      * Skips the test for a given version of PHP.
      */
-    public function skipForVersion(string $version, string $message = ''): TestCall
+    public function skipForVersion(string $version, string $operator = null, string $message = ''): TestCall
     {
         return $this->skip(
-            $this->sameVersion($version),
+            version_compare(PHP_VERSION, $version, $operator ?? '==') === 0,
```

Comment:
When using the 3rd parameter (operator), the `version_compare()` function will return a boolean. 👍🏻 So it doesn't need the `=== 0`. Same with the other one. 🙂

Reply:
Changed that!